xwiki-platform-core/xwiki-platform-administration/xwiki-platform-administration-ui/src/main/resources/XWiki/TemplateProviderSheet.xml

```diff
@@ -42,56 +42,52 @@
   <content>{{velocity}}
 #set ($discard = $xwiki.jsx.use('XWiki.TemplateProviderSheet'))
 #set ($providerObj = $doc.getObject('XWiki.TemplateProviderClass'))
-#if ($editor == 'inline')
-  ; {{translation key="xe.templateprovider.name"/}}
-  : {{html}}&lt;input type="text" value="$!{doc.getTitle()}" name="title" /&gt;{{/html}}
-#end
-; {{translation key="xe.templateprovider.templatename"/}}
-#if ($editor == 'inline')
+{{html wiki="true"}}
```

Comment:
I don`t think waiting will do us any good.

Since you`ve started using the HTML macro, I suggest you use it all the way (i.e. use dd/dt, etc.) so that we can progress on this part faster, since the template provider UI needs some other work.